tests/test_backward_compatibility.py

```diff
@@ -0,0 +1,29 @@
+def test_backward_compatibility():
```

Comment:
Not sure if these capture the failure in py-evm. Exception raised not in when elements are imported but in when functions like `is_on_curve` consume elements.

Reply:
Sorry if I mislead about which test to add. The test should replicate the error detailed in #59 (when run against master).

Reply:
This command helps run all 63 failing json-fixture tests in 30 seconds.
```
py-evm (master)$ pytest tests/json-fixtures/test_blockchain.py  -k 'pairing'
```